README.rst

```diff
@@ -39,7 +39,7 @@ a configuration section that looks like this:
 
 ::
 
-    [[Provider.PluginKaetzchen]]
+    [[Provider.CBORPluginKaetzchen]]
       Capability = "zec"
```

Comment:
I wonder if it should be "eth" instead of "zec"

Reply:
the capability string isn't really used anywhere except in the PKI consensus document and potentially clients can search for a capability string. therefore you should feel free to change it to whatever you want.